pytest.ini

```
@@ -1,3 +1,13 @@
 [pytest]
+addopts =
+    # show tests that (f)ailed, (E)rror, or (X)passed in the summary
+    -rfEX
+    # Make tracebacks shorter
+    --tb=native
+    # enable all warnings
+    -Wd
```

I personally would go a step further

```suggestion
    # turn warnings into error
    -We
```

Right now there aren't any warnings so this is a good time to adopt such a change. With this option set, we get an early heads-up if something is about the change in the future.